docs/views/examples/pass-data/index.html

```diff
@@ -80,9 +80,17 @@ <h3 class="govuk-heading-s">
       <pre class="app-code" tabindex="0"><code>&lt;input type="radio" name="claimant[over-18]" value="yes" {%raw%}{{ checked("['claimant']['over-18']", "yes") }}{%endraw%}&gt;</code></pre>
 
       <h3 class="govuk-heading-s">
-        Setting default data
+        Setting default values
       </h3>
 
+      <p>
+        Default values are pre-filled data in some of your form’s fields. For example, the day’s date on a form that records the date of submission.
+      </p>
+
+      <p>
+        You should set default values to make sure your form’s fields are pre-filled where possible. This saves the user from having to enter data unnecessarily, when you already know something.
```

Comment:
```suggestion
        Where possible, you should set default values to make sure your form’s fields are pre-filled. This saves the user having to enter data unnecessarily, when you already know something.
```

Reply:
Changing the structure of 1st sentence to clarify when to set values info is at the start.

Also a query on the 'when you know something' clause - what does this mean?

Reply:
@NoraGDS Great suggestion! Thanks, will add in.

Hmmm yeah, that second sentence is a bit vague alright. Would it work better if we changed "something" to "the answer"?

Reply:
The confusion for me is that the sentence references 'the user' and then changes to 'you' and it's not clear if it's referring to the same person because the 'you' in the sentence before is the prototype maker. We could try it without the final clause and try either:

-  This saves the user from having to enter information unnecessarily.
- This saves the user from having to enter an answer unnecessarily.

Reply:
Nice! Yep, that makes sense. Shall I add in the following?

```suggestion
        Where possible, you should set default values to make sure your form’s fields are pre-filled. This saves the user from having to enter information unnecessarily.
```

Reply:
👍 Thanks Eoin - yes, that works well.

Reply:
@NoraGDS Cheers! I appreciate your help - looks a heck of a lot better now! 👍🏻